src/main/java/oracle/r2dbc/impl/OracleConnectionFactoryImpl.java

```diff
@@ -59,7 +59,19 @@
  *   <li>{@link ConnectionFactoryOptions#DATABASE}</li>
  *   <li>{@link ConnectionFactoryOptions#USER}</li>
  *   <li>{@link ConnectionFactoryOptions#PASSWORD}</li>
- * </ul>
+ *   <li>{@link ConnectionFactoryOptions#CONNECT_TIMEOUT}</li>
+ *   <li>{@link ConnectionFactoryOptions#SSL}</li>
+ * </ul><p>
+ * A value set for {@code CONNECT_TIMEOUT} will be rounded up to the nearest
```

Comment:
Would it be easier to read to include these comments in the above list, perhaps using a definition list?

Reply:
Thanks to this comment, I've now learned something new about HTML:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl
 
I will use <dl> here, and probably start using it in many other places too.

Reply:
Also, any HTML we type in a comment will be rendered. Now I can try out this new tag:
<dl>
<dt>CONNECT_TIMEOUT</dt>
<dd>The duration will be rounded up to the nearest whole second</dd>
</dl>